src/test/scala/com/fulcrumgenomics/testing/SamRecordSetBuilder.scala

```diff
@@ -102,23 +102,33 @@ class SamRecordSetBuilder(val readLength: Int=100,
     recs
   }
 
-  /** Adds a non-paired read. */
+  /** Adds a non-paired read. Returns an Option for convenience of calling map/foreach etc. */
   def addFrag(name: String = nextName,
-              contig: Int  = 0,
-              start: Int,
+              contig: Int = 0,
+              start: Int = SAMRecord.NO_ALIGNMENT_START,
               unmapped: Boolean = false,
               cigar: String = readLength + "M",
               mapq: Int = 60,
               strand: Strand = Plus,
               baseQuality: Int = this.baseQuality,
-              attrs: Map[String,AnyRef] = Map.empty) : SAMRecord = {
+              attrs: Map[String,AnyRef] = Map.empty) : Option[SAMRecord] = {
```

Comment:
:/ I actually rely on this elsewhere, so changing the return type is a good idea to make it similar to `addPair`, we should be careful about changing things like this.